proto/plan_common.proto

```diff
@@ -185,6 +185,7 @@ message Cardinality {
 // Provide statement-local context, e.g. session info like time zone, for execution.
 message ExprContext {
   string time_zone = 1;
+  uint32 fragment_id = 2;
```

Comment:
I don't think it's a good idea to store the fragment_id here in the ExprContext, since fragment id is independent to expr.

Given that in this PR we have already stored the fragment id in a task local variable, we can directly take the fragment id out from the task local variable when we want to use it (in this PR it's when we create the metric label), instead of taking it out and storing it in expr context and then later reading a field of expr context.

Reply:
I have pushed a commit to fix this. 034a196

Reply:
> I have pushed a commit to fix this. https://github.com/risingwavelabs/risingwave/commit/034a196753242eebc09bcabb706911fd914499f9

Thanks for fix.